### DISCUSSION TITLE
Any suggestions for removing glare from built in led

### DISCUSSION THREAD

**Discussion by JohnnyPicnic** [jomjol/AI-on-the-edge-device#1607] [Q&A] [answered]
I think the glare is causing the middle digits to read incorrectly. 3 as 8, 8 as 3 etc...
I was think of putting some black fabric on the meter where the led seems to reflect the most.
Is there anything else I should try?
![Screen Shot 2022-12-16 at 9 21 44 AM](https://user-images.githubusercontent.com/5553884/208218670-eab9b261-e7aa-4423-833b-460bb61ad5fb.png)

**Comment by caco3** [comment, marked as answer]
Put something as a diffuser in front of the LED, search the discussions here for inspiration.
